CYCLONEDDS_QUALITY_DECLARATION.md

```diff
@@ -2,9 +2,9 @@ This document is a declaration of software quality for the `CycloneDDS` external
 
 # `CycloneDDS` Quality Declaration
```

Comment:
The correct form is "Eclipse Cyclone DDS", which should be spelled out here in full. It can then be abbreviated to Cyclone DDS in the rest of the document.